code/ship/ship.cpp

```diff
@@ -1452,6 +1458,9 @@ ship_info::ship_info()
 	death_roll_r_mult = 1.0f;
 	death_fx_r_mult = 1.0f;
 	death_roll_time_mult = 1.0f;
+	death_roll_xrotation_cap = 0.75f*DEATHROLL_ROTVEL_CAP;
+	death_roll_yrotation_cap = 0.75f*DEATHROLL_ROTVEL_CAP;
```

Comment:
If this constant is only ever used after being multiplied by 0.75, why is that multiplication not part of that constant's value? I.e. Why is it 6.3 and not 4.725?

Reply:
Unfortunately there's [one other place](https://github.com/scp-fs2open/fs2open.github.com/pull/2030/files#diff-d0fc37f76f2dd8a965a41def714ac5bbL1485) where the constant is used as is, i.e. without being multiplied by 0.75.

Reply:
Okay, fair enough